[
  {
    "DefinitionID": "oval:com.ubuntu.trusty:def:202082770000000",
    "Title": "CVE-2020-8277 on Ubuntu 14.04 LTS (trusty) - medium.",
    "Description": "A Node.js application that allows an attacker to trigger a DNS request for a host of their choice could trigger a Denial of Service in versions \u003c 15.2.1, \u003c 14.15.1, and \u003c 12.19.1 by getting the application to resolve a DNS record with a larger number of responses. This is fixed in 15.2.1, 14.15.1, and 12.19.1.",
    "Advisory": {
      "Severity": "Medium",
      "Cves": [
        {
          "CveID": "CVE-2020-8277",
          "Cvss2": "",
          "Cvss3": "",
          "Cwe": "",
          "Impact": "",
          "Href": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-8277",
          "Public": ""
        }
      ],
      "Bugzillas": [],
      "AffectedCPEList": [],
      "Issued": "1000-01-01T00:00:00Z",
      "Updated": "1000-01-01T00:00:00Z"
    },
    "Debian": null,
    "AffectedPacks": [],
    "References": [
      {
        "Source": "CVE",
        "RefID": "CVE-2020-8277",
        "RefURL": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-8277"
      },
      {
        "Source": "Ref",
        "RefID": "",
        "RefURL": "http://people.canonical.com/~ubuntu-security/cve/2020/CVE-2020-8277.html"
      },
      {
        "Source": "Ref",
        "RefID": "",
        "RefURL": "https://nodejs.org/en/blog/vulnerability/november-2020-security-releases/#denial-of-service-through-dns-request-cve-2020-8277"
      },
      {
        "Source": "Ref",
        "RefID": "",
        "RefURL": "https://github.com/c-ares/c-ares/commit/0d252eb3b2147179296a3bdb4ef97883c97c54d3"
      },
      {
        "Source": "Ref",
        "RefID": "",
        "RefURL": "https://ubuntu.com/security/notices/USN-4638-1"
      }
    ]
  }
]
